go/host/p2p/p2p.go

```diff
@@ -77,7 +77,7 @@ func (p *p2pImpl) UpdatePeerList(newPeers []string) {
 }
 
 func (p *p2pImpl) BroadcastTx(tx common.EncryptedTx) {
-	p.broadcast(Tx, tx, append(p.peerAddresses, p.ourAddress))
```

Comment:
As we're processing the tx first ourselves, we no longer need to send it to ourselves for processing.